phonecall_plan/wizards/phonecall_plan_view.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--© 2016 6IT
+	License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
+	<data>
+		<record id="add_call_view_form" model="ir.ui.view" >
+			<field name="name">Scheduled calls</field>
+			<field name="model">add.call</field>
+			<field name="arch" type="xml">
+				<form>
```

Comment:
I have 4 spaces per indent. Do you think it is too much? Should I use 2 instead ?